geopandas/tests/test_indexing.py

```diff
@@ -0,0 +1,72 @@
+import pytest
```

Comment:
I moved these tests to a new file. Originally they were with test_indexing in test_pandas_methods before I rewrote them as parametrised. I still thought it might be a distraction to have it there, but happy to move back.

Reply:
They might also fit in the `test_op_output_types.py`?